db/migrate/20200526195717_adding_publish_target_type.rb

```diff
@@ -0,0 +1,10 @@
+class AddingPublishTargetType < ActiveRecord::Migration[6.0]
+  def change
+    add_column :publish_targets, :target_type, :string, null: false, default: PublishTarget::Type::PRODUCTION
```

Comment:
I'm not sure we want to have a default value for target_type.  It seems like the kind of thing where users should be deliberate about choosing the target_type, and a validation on the required field would catch cases where it wasn't supplied.

Reply:
I think I provided a default because one HAS to be provided if the field cannot be null. That should probably double checked, but I remember being forced to add that in.

Reply:
@cgalarza Ah, right!  Makes sense.